gcloud/orb.yaml

```diff
@@ -350,7 +351,7 @@ jobs:
           Name of dockerfile to use.
         type: string
       executor:
-        default: docker:18.06.3-ce-git
+        default: docker:18.09.9-git
```

Comment:
Should this match the `18.09.3` you're requesting above? May not matter, I suppose?

Reply:
Strangely, the "real" version seems to be the one under `setup_remote_docker`. But yeah, it's nice to have them aligned

Reply:
So if I understand correctly, the executor would define the image you are running commands within -- in other words, the version of the docker _client_ you're using. The `setup_remote_docker` command would be for configuring the _server_. Theoretically, clients should be backwards compatible with virtually all servers, but I'm not sure that's an explicit guarantee (especially now that they've switched from semver to calver.. :().